app/(admin)/admin/notifications/page.tsx

```diff
@@ -1,16 +1,71 @@
 // Copyright (c) Gridiron Survivor.
 // Licensed under the MIT License.
 
-import { JSX } from 'react';
+'use client';
+import { Button } from '@/components/Button/Button';
+import { getCurrentLeague } from '@/api/apiFunctions';
+import { Input } from '@/components/Input/Input';
+import { JSX, useState } from 'react';
+import { Label } from '@/components/Label/Label';
+import { sendEmailNotifications } from './actions/sendEmailNotification';
+import React from 'react';
 
 /**
  * The admin home page.
  * @returns The rendered AdminHome page.
  */
 const AdminNotifications = (): JSX.Element => {
+  const [content, setContent] = useState<string>('');
+  const [sendEmailUsers, setSendEmailUsers] = useState<string[]>([]);
+  const [subject, setSubject] = useState<string>('');
+
+  /**
+   * To grab all participant's userIDs from the league.
+   */
+  const participantsEmail = async (): Promise<void> => {
```

Comment:
If we're grabbing the participants' userIDs from the league with this function, is calling it `participantsEmail` correct?

Reply:
yes. the function sends an email to all participants.